### DISCUSSION TITLE
MuPDF 0.2.3

### DISCUSSION THREAD

**Discussion by jamie-lemon** [ArtifexSoftware/mupdf.js#103] [Announcements]
We just released version 0.2.3 on NPM - this fixes the `setMetaData` method for annotations.
See: https://www.npmjs.com/package/mupdf?activeTab=readme

**Comment by mfts** [comment]
Awesome @jamie-lemon 🙌

Do you plan to use GitHub releases for releasing tagged versions?

**Reply by jamie-lemon** [reply]
Good point - have now started! :)
